test/integration/tcp_tunneling_integration_test.cc

```diff
@@ -271,12 +286,7 @@ TEST_P(ConnectTerminationIntegrationTest, BuggyHeaders) {
   // Sending a header-only request is probably buggy, but rather than having a
   // special corner case it is treated as a regular half close.
   codec_client_ = makeHttpConnection(lookupPort("http"));
-  response_ = codec_client_->makeHeaderOnlyRequest(
-      Http::TestRequestHeaderMapImpl{{":method", "CONNECT"},
-                                     {":path", "/"},
-                                     {":protocol", "bytestream"},
-                                     {":scheme", "https"},
-                                     {":authority", "foo.lyft.com:80"}});
+  response_ = codec_client_->makeHeaderOnlyRequest(getRequestHeaderMap());
```

Comment:
can't this just always be regular request headers?

Reply:
Done